internal/cli/dbusers/create.go

```diff
@@ -231,6 +244,15 @@ func CreateBuilder() *cobra.Command {
 	cmd.Flags().StringVar(&opts.x509Type, flag.X509Type, none, usage.X509Type)
 	cmd.Flags().StringVar(&opts.awsIamType, flag.AWSIAMType, none, usage.AWSIAMType)
 	cmd.Flags().StringVar(&opts.ldapType, flag.LDAPType, none, usage.LDAPType)
+	cmd.Flags().StringVar(&opts.oidcType, flag.OIDCType, none, usage.OIDCType)
+
+	cmd.MarkFlagsMutuallyExclusive(flag.AWSIAMType, flag.LDAPType)
+	cmd.MarkFlagsMutuallyExclusive(flag.AWSIAMType, flag.X509Type)
+	cmd.MarkFlagsMutuallyExclusive(flag.AWSIAMType, flag.OIDCType)
+	cmd.MarkFlagsMutuallyExclusive(flag.LDAPType, flag.X509Type)
+	cmd.MarkFlagsMutuallyExclusive(flag.LDAPType, flag.OIDCType)
+	cmd.MarkFlagsMutuallyExclusive(flag.X509Type, flag.OIDCType)
+	cmd.MarkFlagsMutuallyExclusive(flag.Password, flag.OIDCType)
```

Comment:
I think this is valid
```suggestion
	cmd.MarkFlagsMutuallyExclusive(flag.AWSIAMType, flag.LDAPType, flag.X509Type,ag.OIDCType)
	cmd.MarkFlagsMutuallyExclusive(flag.LDAPType, flag.X509Type)
	cmd.MarkFlagsMutuallyExclusive(flag.LDAPType, flag.OIDCType)
	cmd.MarkFlagsMutuallyExclusive(flag.X509Type, flag.OIDCType)
	cmd.MarkFlagsMutuallyExclusive(flag.Password, flag.OIDCType)
```
but the documentation is not clear if it would work the same, have you tried?

Reply:
`Error: if any flags in the group [ldapType oidcType] are set none of the others can be; [ldapType oidcType] were all set`